### ISSUE TITLE
base64 bindings

### ISSUE THREAD

**Issue by naturallymitchell** [patchworkuniversity/torchbear#180]
https://github.com/alicemaz/rust-base64

**Comment by mitchtbaum** [comment]
for https://github.com/foundpatterns/lighttouch-base/issues/111#issuecomment-449542353
base64.to_string() and base64.from_string() ?
//cc @Arnaz87

**Comment by dariusc93** [comment]
rename them from encode/decode to from_string/to_string?

**Comment by mitchtbaum** [comment]
let's see what Arnaud says first

**Comment by mitchtbaum** [comment]
it has to be base64.whatever, right?

**Comment by mitchtbaum** [comment]
not to_base64 / from_base64, base64.from_string and base64.to_string

**Comment by Arnaz87** [comment]
Not necessarily, it can be like the current lua library, a global to_base64 function. I still like encode more but this is much much better

**Comment by mitchtbaum** [comment]
which is much much better?

you like the current base64.encode / base64.decode ?

**Comment by Arnaz87** [comment]
Yes, I like more `base64.encode` because it's in it's own module, but just `to_base64` is also a nice option and it's shorter to write, `base64.to_base64` is just ugly.

**Comment by mitchtbaum** [comment]
`base64.to_base64` was never an option

it's either `base64.from_string`, `base64.encode`, or `to_base64`

**Comment by mitchtbaum** [comment]
https://github.com/foundpatterns/lighttouch-base/issues/111 depends on this

**Comment by Arnaz87** [comment]
I very much dislike from/to_string, because both the output and input are strings so the names are very useless to distinguish between them, but decode and encode are pretty clear given the definitions of those words, I prefer those by far.

**Comment by dariusc93** [comment]
Are there anymore comments regarding the naming to base64 or can this be closed?

**Comment by mitchtbaum** [comment]
cool
